### DISCUSSION TITLE
Contract verification fails on Goerli but passes just fine on Sepolia.

### DISCUSSION THREAD

**Discussion by 0xFloop** [foundry-rs/foundry#3768] [Help]
Version:
forge 0.2.0 (6871421 2022-11-25T00:15:01.855568Z)

Issue:
My contract verifications only fail no matter if I use forge create, forge verify-contract, or a deploying script with --verify. The same exact bytecode in artifacts as etherscan so I'm very confused. Same code and commands work just fine for verifying on Sepolia so I'm wondering if its an issue with etherscans Goerli verification api. Here is a repo with contract that fails to verify on Goerli but verifies just fine on Sepolia [HERE](https://github.com/0xFloop/failed_verification_repo).

**Comment by appleseed-iii** [comment]
your `etherscan` section of `foundry.toml` is missing sepolia
<img width="375" alt="image" src="https://user-images.githubusercontent.com/80423742/211557490-19ffa4d8-0f82-4002-8365-250470744e35.png">
